pages/index.tsx

```diff
@@ -1,19 +1,39 @@
+import { WelcomeToSDC } from "@components/WelcomeToSDC";
 import { useSession, signIn, signOut } from "next-auth/client";
+import { ProgramList } from "../src/components/ProgramList";
+import { Box, Flex, Divider, Spacer, Heading } from "@chakra-ui/react";
 
 export default function Component() {
-    const [session, loading] = useSession();
-    if (session) {
-        return (
-            <>
-                Signed in as {session.user.email} <br />
-                <button onClick={() => signOut()}>Sign out</button>
-            </>
-        );
-    }
+    // const [session, loading] = useSession();
+    // if (session) {
+    //     return (
+    //         <>
+    //             Signed in as {session.user.email} <br />
+    //             <button onClick={() => signOut()}>Sign out</button>
+    //         </>
+    //     );
+    // }
```

Comment:
can we move this to a diff page just so that we don't lose the login flow?

Reply:
i moved it, made a programs.tsx file